src/monexamine.cpp

```diff
@@ -162,6 +211,52 @@ int monexamine::pet_armor_pos( monster &z )
     return pos;
 }
 
+void monexamine::attach_or_remove_saddle( monster &z )
+{
+    if( z.has_effect( effect_saddled ) ) {
+        z.remove_effect( effect_saddled );
+        item riding_saddle( "riding_saddle", 0 );
+        g->u.i_add( riding_saddle );
+    } else {
+        z.add_effect( effect_saddled, 1_turns, num_bp, true );
+        g->u.use_amount( "riding_saddle", 1 );
+    }
+}
+
+void monexamine::mount_pet( monster &z )
+{
+    g->u.add_effect( effect_riding, 1_turns, num_bp, true );
+    z.add_effect( effect_ridden, 1_turns, num_bp, true );
+    if( z.has_effect( effect_tied ) ) {
+        z.remove_effect( effect_tied );
+        item rope_6( "rope_6", 0 );
+        g->u.i_add( rope_6 );
+    }
+    if( z.has_effect( effect_harnessed ) ) {
+        z.remove_effect( effect_harnessed );
+        add_msg( m_info, _( "You remove the %s's harness" ), z.get_name() );
+    }
+    z.add_carried_creature( g->u );
+    tripoint pnt = z.pos();
+    auto mons = g->critter_tracker->find( pnt );
+    if( mons == nullptr ) {
+        add_msg( m_debug, _( "mount_pet() : monster not found in critter_tracker" ) );
```

Comment:
```suggestion
        add_msg( m_debug, "mount_pet() : monster not found in critter_tracker" );
```
No need to translate debug messages.